recipes/RingsPy/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "RingsPy" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ebd262c73994c983ce7aaffff441dfc97f5930050c0e4ba0f0f2eb8253d12afe
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - setuptools
+    - wheel
+    - requests >=2
+    - numpy
+    - scipy
+    - matplotlib >=3.2.2
```

Unless you need a pyqt-based gui, it's better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base >=3.2.2
```

---

@dopplershift Thanks for the comments. I have modified it correspondingly.